email/templates/marketing/ngan-fundmethod.html

```diff
@@ -0,0 +1,330 @@
+                                <p style="padding: 0px;font-family: 'IBM Plex Sans', Arial, sans-serif; font-size: 32px; line-height: 1.25 ; color: #ffffff; font-weight: 600;">A new way to</p>
+                                <p style="padding: 0px;font-family: 'IBM Plex Sans', Arial, sans-serif; font-size: 32px; line-height: 1.25 ; color: #ffffff; font-weight: 600;">fund your account</p>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td bgcolor="#eeeeee" align="center"  >
+                                <img src="https://static.deriv.com/email/images/marketing/dragonpay.png" width="600" height="307" border="0" class="max-width" style=" display: block;" alt="Deriv.com">
+                            </td>
+                        </tr>
+                        <!-- COPY -->                      
+                        <tr>
+                            <td bgcolor="#ffffff" align="left" style="padding: 30px 30px 25px 30px;">
+                                <p style="font-family: 'IBM Plex Sans', Arial, sans-serif; color: #333333; font-size: 16px; font-weight: 400; line-height: 1.5; margin: 10px 0px 0px 0px;">We’re pleased to let you know that Ngan Luong is now available as a payment option on Deriv.com.</p>
+                                <p style="font-family: 'IBM Plex Sans', Arial, sans-serif; color: #333333; font-size: 16px; font-weight: 400; line-height: 1.5; margin: 18px 0px 0px 0px;">With this new partnership, you and your clients can now deposit and withdraw funds from Deriv.com to your domestic bank accounts. To learn how to use Ngan Luong for deposits and withdrawals, download the PDF below.</p>
+                            </td>
+                        </tr>
+                        <!-- button -->
+                        <tr>
+                            <td bgcolor="#ffffff" align="center" style="padding: 0px 30px 0px; color: #333333; font-family: 'IBM Plex Sans', Arial, sans-serif; font-size: 16px; font-weight: 400; line-height: 24px;">
+                                <!--[if (gte mso 9)|(IE)]>
+                                    <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="https://deriv.com/payment-methods/nganluong-payment-method.pdf" style="height:40px;v-text-anchor:middle;width:220px;" arcsize="8%" stroke="f" fillcolor="#ff444f">
+                                    <w:anchorlock/>
+                                    <center style="color:#ffffff;font-family:sans-serif;font-size:16px;font-weight:bold;">Download PDF</center>
+                                    </v:roundrect>
+                                <![endif]-->
+                                <a class="button-secondary" href="https://deriv.com/payment-methods/nganluong-payment-method.pdf" style="mso-hide:all;"><span>Download PDF</span></a>
```

Comment:
remove span from link 
```suggestion
                                <a class="button-secondary" href="https://deriv.com/payment-methods/nganluong-payment-method.pdf" style="mso-hide:all;">Download PDF</a>
```

Reply:
Done